using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md

```diff
@@ -1,5 +1,82 @@
 # Mobile Core API Reference
 
+# Application reference
+
+When building Android applications, it is required to pass the `android.app.Application` reference to the Mobile SDK, which allows the Mobile SDK to access the `android.app.Context` and monitor the lifecycle of the Andorid application.
```

Comment:
Please replace "it is required to" with "the `android.app.Application` reference must be passed to..."